Led.cpp

```diff
@@ -1,33 +1,51 @@
 #include "Led.h"
 
-Led::Led() : pin(pin) {
+Led::Led() : Pin(Pin) {
```

Comment:
the `Pin` inside the parenthesis are undefined because no parameter is passed in. Let default pin to 0 if no value is given.